components/builder-index/EnablingConditionsSelector.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import classnames from 'classnames';
+
+class EnablingConditionsSelector extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      activeTab: "stakeholder-actor",
+    };
+  }
+
+  selectTab(key) {
+    this.setState({ activeTab: key });
+  }
+
+  render() {
+    const tabs = {
+      actor: "Actor",
```

Comment:
Oops, that's dead code. I did that initially just to make the layout work, but eventually moved on to using the results from the API. I'm not using that variable `tabs` anywhere in the code. Deleting it.